nav2_controller/src/nav2_controller.cpp

```diff
@@ -195,8 +195,12 @@ ControllerServer::on_configure(const rclcpp_lifecycle::State & state)
 
   // Create the action server that we implement with our followPath method
   action_server_ = std::make_unique<ActionServer>(
-    rclcpp_node_, "follow_path",
-    std::bind(&ControllerServer::computeControl, this));
+    shared_from_this(),
+    "follow_path",
+    std::bind(&ControllerServer::computeControl, this),
+    nullptr,
+    std::chrono::milliseconds(500),
```

Comment:
I didn't ask in the Planner PR and I should have -- is this 500ms the same behavior as before or did this value change?

Reply:
before PR, the default value was used, and 500ms is the default value, so it's same as before.